3_visualize/src/03_draw_plot.R

```diff
@@ -1,6 +1,7 @@
-# this function returns the evaulated data plots
+# this function returns the evaluated data plots
 
-draw_plot = function(eval_data, directory) {
+draw_plot = function(data, directory) {
```

Comment:
Here, too! Instead of `directory`, try `out_file` to be consistent with the previous functions. Last thing is that you don't need to wrap them in `file.path()` if it is already a complete filepath. You can just write `png(file = out_file, width = 8 ...`

Reply:
Updated file name and erased file.path().